### ISSUE TITLE
Settings not saving

### ISSUE THREAD

**Issue by MarioGuy588** [DS-Homebrew/TWiLightMenu#871]
**Your console configuration:** DSi with HiyaCFW and Unlaunch version 1.9
**TWiLight Menu++ Version:** Release version 11.2.0
**TWiLight Menu++ Theme:** 3DS
### Expected behaviour

When I go into the twilight menu settings I should be able to change a setting and have it save. For example, changing my theme from 3DS to Saturn, or Changing GBARunner2 to use ARM7 instead of ARM9.

### Actual behaviour

Whenever I change a setting in the twilight menu settings, the console doesn't save them and instead reverts back to the previous settings

### Steps to reproduce

1. Upgraded from a previous Twilight menu version to 11.2.0
2. Go into settings to change a setting

**Comment by RocketRobz** [comment]
Make sure the SD card switch isn't set to lock.

**Comment by MarioGuy588** [comment]
I've booted the system up with the sd card both locked and unlocked and I still get the same result. I should also add that I can't load any other games besides New Super Mario Bros. Even when I highlight a different game to boot up it loads New Super Mario Bros.

**Comment by RocketRobz** [comment]
Delete both `settings.ini` and `nds-bootstrap.ini` at `sd:/_nds/TWiLightMenu/` and `sd:/_nds/`.

**Comment by MarioGuy588** [comment]
I just tried this twice and I'm still having the same problem

**Comment by RocketRobz** [comment]
Check your SD card for errors.

**Comment by MarioGuy588** [comment]
No errors found.

**Comment by RocketRobz** [comment]
What tool did you use?

**Comment by MarioGuy588** [comment]
The default windows scan and repair tool. Is there another tool I should be using?

**Comment by RocketRobz** [comment]
Nah, it's what I use. :P

So after deleting said files, does NSMB still boot?

**Comment by MarioGuy588** [comment]
Yes. When I booted up the system after deleting the files, I selected a game that wasn't New Super Mario Bros., in my case Namco Museum, it says on the bottom screen "creating RAM dump file. Please wait." Then it loads NDS bootstrap and New Super Mario Bros.

**Comment by MarioGuy588** [comment]
This happened both times btw.

**Comment by RocketRobz** [comment]
Try reformatting the SD card.

**Comment by MarioGuy588** [comment]
I tried twice and it said "Windows was unable to complete the format"

**Comment by RocketRobz** [comment]
Yep, something's wrong with the SD card. Could be dying.
You'll need to get another SD card.

**Comment by MarioGuy588** [comment]
;/ Oh well. Thanks for your help anyways.

**Comment by MarioGuy588** [comment]
Okay, I confirmed it was the SD card. I copied all of the files to a new card and the new card worked. Thanks again for your help with this.

**Comment by renvqi** [comment]
I was also having issues with settings not saving. my case, I always exit the twilightmenu++ settings by pressing power button which restarts the dsi (not sure if this is proper way lol) so idk if this is why it doesn't save.

anyway, my settings changes would never save when exiting settings via pressing "power button". for some reason I decided to press a+b one time while trying different combo to see if there was actually a control I had to use to save the changes. this also restarted the dsi but I found out by doing this the changes I made were now saved.
